plugins/modules/grafana_dashboard.py

```diff
@@ -507,6 +508,10 @@ def main():
     module = AnsibleModule(
         argument_spec=argument_spec,
         supports_check_mode=False,
+        required_if=[
+            ['state', 'export', ['path']],
+            ['state', 'present', ['path']],
```

Comment:
The CI failed because in this collection, the module supports installing a dashboard from an url or a grafana.com dashboard id.

So the `path` is no more the only option when `state` is `present`.
The tests using the dashboard id fails with the line 513 constraint. It should pass the CI without this line.

Reply:
Is it possible to use "or statement" there?